pkg/bpf/bpffs.go

```diff
@@ -129,6 +175,35 @@ func mountFS() error {
 		if err != nil {
 			return fmt.Errorf("Command execution failed: %s\n%s", err, out)
 		}
+	} else { // Already mounted. We need to fail if mounted multiple times.
+
+		// Execute the following command to find multiple bpffs mount points
+		// % mount | grep "<mapRoot> " | wc -l | cut -f1 -d' '
+		newmapRoot := mapRoot + " " // Append space to ignore /bpf/xdp and /bpf/ip mountpoints.
+		mt := exec.Command("mount")
+		grep := exec.Command("grep", newmapRoot)
+		wc := exec.Command("wc", "-l")
+		cut := exec.Command("cut", "-f1", "-d ")
+
+		output, stderr, _ := mountCmdPipe(mt, grep, wc, cut)
+
+		if len(stderr) > 0 {
+			return fmt.Errorf("Command execution failed: %s\n", stderr)
```

Comment:
When returning errors the error message should start with small case so the function caller can use it on a sentence, for example:

```
err := mountFs()
if err != nil{
    log.Errorf("Something failed while mounting: %s", err)
}
```
As it is, the log will print
```
Something failed while mounting: Command execution failed: ..
```
instead of 
```
Something failed while mounting: command execution failed: ..
```

Also, remove the trailing `\n`

Reply:
will do

Reply:
Done